node/core/approval-voting/src/tests.rs

```diff
@@ -620,12 +620,12 @@ async fn check_and_import_assignment(
 		overseer,
 		FromOrchestra::Communication {
 			msg: ApprovalVotingMessage::CheckAndImportAssignment(
-				IndirectAssignmentCert {
+				IndirectAssignmentCertV2 {
 					block_hash,
 					validator,
 					cert: garbage_assignment_cert(AssignmentCertKind::RelayVRFModulo { sample: 0 }),
```

Comment:
or yes it could be set to zero and the comment could mark it as depricated if we do not want to change the wire format.